CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [7.2.1]
+
+### Added
+* Added `HyP3.submit_insar_isce_multi_burst_job` to README example
```

Comment:
```suggestion
* Added `hyP3.submit_insar_isce_multi_burst_job` to README example
```

Reply:
https://github.com/ASFHyP3/hyp3-sdk/blob/86961045a318a7302e9406ba24d50018c5998d5d/src/hyp3_sdk/hyp3.py#L21C1-L21C12

This based on this class. Also other changelog entries use HyP3